### PR TITLE
DBZ-5484 Better error reporting in SKIP/WARN cases

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-5484

### CODE REVIEW

File: debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java
Comment (jpechane) [inline debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java:252]: We don't want to throw new exception. We just want to modify this logging message and the one below to contain the full exception.

File: debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java
Comment (jpechane) [inline debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java:251]: You should not remove the `break` statements. Also could you please try to reproduce the situation and post here from the log how the new error message looks like? Thanks a lot

**Review by Naros — APPROVED** [review]
LGTM now, once CI finishes I'll merge.  Thanks @ani-sha.

**Pull request merged**
